### PR TITLE
[MINOR] update dev/create-release/known_translations

### PR DESCRIPTION
**Pull request by cloud-fan**
This is auto-updated by running script `translate-contributors.py`

### CODE REVIEW

**Comment by cloud-fan** [comment]
cc @gatorsmile @dongjoon-hyun

**Review by srowen — COMMENTED** [review]
You can sort the file with `sort` but it doesn't matter.

**Review by srowen — APPROVED** [review]

**Comment by cloud-fan** [comment]
looks like jenkins is unstable again...

Anyway this PR doesn't need to go through jenkins. I'm merging it to master, thanks!

**Review by dongjoon-hyun — COMMENTED** [review]
+1, late LGTM. Thanks.

File: dev/create-release/known_translations [hunk 1]
```diff
@@ -261,3 +261,151 @@ yanlin-Lynn - Yanlin Wang
...
+triplesheep - triplesheep
+turboFei - Fei Wang
+ulysses-you - ulysses-you
+uzadude - Ohad Raviv
+wackxu - wackxu
+wangjiaochun - wangjiaochun
+wangshisan - wangshisan
+weixiuli - XiuLi Wei
+wenfang6 - wenfang6
+wenxuanguan - wenxuanguan
```
Comment (cloud-fan) [inline dev/create-release/known_translations:405]: @wenxuanguan can you let us know your full name?

File: dev/create-release/known_translations [hunk 2]
```diff
@@ -261,3 +261,151 @@ yanlin-Lynn - Yanlin Wang
...
+tools4origins - tools4origins
+triplesheep - triplesheep
+turboFei - Fei Wang
+ulysses-you - ulysses-you
+uzadude - Ohad Raviv
+wackxu - wackxu
+wangjiaochun - wangjiaochun
+wangshisan - wangshisan
+weixiuli - XiuLi Wei
+wenfang6 - wenfang6
```
Comment (cloud-fan) [inline dev/create-release/known_translations:404]: @wenfang6 can you let us know your full name?

File: dev/create-release/known_translations [hunk 3]
```diff
@@ -261,3 +261,151 @@ yanlin-Lynn - Yanlin Wang
...
+teeyog - teeyog
+tinhto-000 - Tin Hang To
+tools4origins - tools4origins
+triplesheep - triplesheep
+turboFei - Fei Wang
+ulysses-you - ulysses-you
+uzadude - Ohad Raviv
+wackxu - wackxu
+wangjiaochun - wangjiaochun
+wangshisan - wangshisan
```
Comment (cloud-fan) [inline dev/create-release/known_translations:402]: @wangshisan can you let us know your full name?

File: dev/create-release/known_translations [hunk 4]
```diff
@@ -261,3 +261,151 @@ yanlin-Lynn - Yanlin Wang
...
+suxingfate - Xinglong Wang
+teeyog - teeyog
+tinhto-000 - Tin Hang To
+tools4origins - tools4origins
+triplesheep - triplesheep
+turboFei - Fei Wang
+ulysses-you - ulysses-you
+uzadude - Ohad Raviv
+wackxu - wackxu
+wangjiaochun - wangjiaochun
```
Comment (cloud-fan) [inline dev/create-release/known_translations:401]: @wangjiaochun can you let us know your full name?

File: dev/create-release/known_translations [hunk 5]
```diff
@@ -261,3 +261,151 @@ yanlin-Lynn - Yanlin Wang
...
+sujith71955 - Sujith Chacko
+suxingfate - Xinglong Wang
+teeyog - teeyog
+tinhto-000 - Tin Hang To
+tools4origins - tools4origins
+triplesheep - triplesheep
+turboFei - Fei Wang
+ulysses-you - ulysses-you
+uzadude - Ohad Raviv
+wackxu - wackxu
```
Comment (cloud-fan) [inline dev/create-release/known_translations:400]: @wackxu can you let us know your full name?

File: dev/create-release/known_translations [hunk 6]
```diff
@@ -261,3 +261,151 @@ yanlin-Lynn - Yanlin Wang
...
+squito - Imran Rashid
+stczwd - Jackey Lee
+sujith71955 - Sujith Chacko
+suxingfate - Xinglong Wang
+teeyog - teeyog
+tinhto-000 - Tin Hang To
+tools4origins - tools4origins
+triplesheep - triplesheep
+turboFei - Fei Wang
+ulysses-you - ulysses-you
```
Comment (cloud-fan) [inline dev/create-release/known_translations:398]: @ulysses-you can you let us know your full name?

File: dev/create-release/known_translations [hunk 7]
```diff
@@ -261,3 +261,151 @@ yanlin-Lynn - Yanlin Wang
...
+slamke - Sun Ke
+southernriver - Liang Chen
+squito - Imran Rashid
+stczwd - Jackey Lee
+sujith71955 - Sujith Chacko
+suxingfate - Xinglong Wang
+teeyog - teeyog
+tinhto-000 - Tin Hang To
+tools4origins - tools4origins
+triplesheep - triplesheep
```
Comment (cloud-fan) [inline dev/create-release/known_translations:396]: @triplesheep can you let us know your full name?

File: dev/create-release/known_translations [hunk 8]
```diff
@@ -261,3 +261,151 @@ yanlin-Lynn - Yanlin Wang
...
+sitegui - Guilherme Souza
+slamke - Sun Ke
+southernriver - Liang Chen
+squito - Imran Rashid
+stczwd - Jackey Lee
+sujith71955 - Sujith Chacko
+suxingfate - Xinglong Wang
+teeyog - teeyog
+tinhto-000 - Tin Hang To
+tools4origins - tools4origins
```
Comment (cloud-fan) [inline dev/create-release/known_translations:395]: @tools4origins can you let us know your full name?

File: dev/create-release/known_translations [hunk 9]
```diff
@@ -261,3 +261,151 @@ yanlin-Lynn - Yanlin Wang
...
+sheepstop - Ting Yang
+shivsood - Shiv Prashant Sood
+sitegui - Guilherme Souza
+slamke - Sun Ke
+southernriver - Liang Chen
+squito - Imran Rashid
+stczwd - Jackey Lee
+sujith71955 - Sujith Chacko
+suxingfate - Xinglong Wang
+teeyog - teeyog
```
Comment (cloud-fan) [inline dev/create-release/known_translations:393]: @teeyog can you let us know your full name?

File: dev/create-release/known_translations [hunk 10]
```diff
@@ -261,3 +261,151 @@ yanlin-Lynn - Yanlin Wang
...
+redsk - Nicola Bova
+roland1982 - roland1982
+rongma1997 - Rong Ma
+rrusso2007 - Rob Russo
+samsetegne - Samuel L. Setegne
+sangramga - Sangram Gaikwad
+sarthfrey - Sarth Frey
+seayoun - Haiyang Yu
+sev7e0 - Jiaqi Li
+shahidki31 - Shahid
```
Comment (cloud-fan) [inline dev/create-release/known_translations:382]: @shahidki31 can you let us know your full name?

File: dev/create-release/known_translations [hunk 11]
```diff
@@ -261,3 +261,151 @@ yanlin-Lynn - Yanlin Wang
...
+oleg-smith - Oleg Kuznetsov
+ozancicek - Ozan Cicekci
+pengbo - Peng Bo
+planga82 - Pablo Langa Blanco
+praneetsharma - Praneet Sharma
+ptkool - Michael Styles
+qb-tarushg - Tarush Grover
+redsanket - Sanket Reddy
+redsk - Nicola Bova
+roland1982 - roland1982
```
Comment (cloud-fan) [inline dev/create-release/known_translations:374]: @roland1982 can you let us know your full name?

File: dev/create-release/known_translations [hunk 12]
```diff
@@ -261,3 +261,151 @@ yanlin-Lynn - Yanlin Wang
...
+lipzhu - Lipeng Zhu
+liucht-inspur - liucht-inspur
+liupc - Pengcheng Liu
+liwensun - Liwen Sun
+manuzhang - Manu Zhang
+mareksimunek - Marek Simunek
+masa3141 - Masahiro Kazama
+mdianjun - Dianjun Ma
+merrily01 - Ruilei Ma
+mob-ai - mob-ai
```
Comment (cloud-fan) [inline dev/create-release/known_translations:360]: @mob-ai can you let us know your full name?

File: dev/create-release/known_translations [hunk 13]
```diff
@@ -261,3 +261,151 @@ yanlin-Lynn - Yanlin Wang
...
+dlindelof - David Lindelof
+dongjoon-hyun - Dongjoon Hyun
+eatoncys - eatoncys
+fan31415 - Yijie Fan
+fitermay - Yuli Fiterman
+francis0407 - Mingcong Han
+fuwhu - Fuwang Hu
+gss2002 - Greg Senia
+hddong - Dongdong Hong
+hehuiyuan - hehuiyuan
```
Comment (cloud-fan) [inline dev/create-release/known_translations:332]: @hehuiyuan can you let us know your full name?

File: dev/create-release/known_translations [hunk 14]
```diff
@@ -261,3 +261,151 @@ yanlin-Lynn - Yanlin Wang
...
+daviddingly - Xiaoyuan Ding
+davidvrba - David Vrba
+deepyaman - Deepyaman Datta
+denglingang - Lingang Deng
+dengziming - dengziming
+deshanxiao - deshanxiao
+dima-asana - Dima Kamalov
+dlindelof - David Lindelof
+dongjoon-hyun - Dongjoon Hyun
+eatoncys - eatoncys
```
Comment (cloud-fan) [inline dev/create-release/known_translations:325]: @eatoncys can you let us know your full name?

File: dev/create-release/known_translations [hunk 15]
```diff
@@ -261,3 +261,151 @@ yanlin-Lynn - Yanlin Wang
...
+colinmjj - Colin Ma
+cxzl25 - cxzl25
+cyq89051127 - Yongqiang Chai
+darrentirto - Darren Tirto
+daviddingly - Xiaoyuan Ding
+davidvrba - David Vrba
+deepyaman - Deepyaman Datta
+denglingang - Lingang Deng
+dengziming - dengziming
+deshanxiao - deshanxiao
```
Comment (cloud-fan) [inline dev/create-release/known_translations:321]: @deshanxiao can you let us know your full name?

File: dev/create-release/known_translations [hunk 16]
```diff
@@ -261,3 +261,151 @@ yanlin-Lynn - Yanlin Wang
...
+codeborui - codeborui
+colinmjj - Colin Ma
+cxzl25 - cxzl25
+cyq89051127 - Yongqiang Chai
+darrentirto - Darren Tirto
+daviddingly - Xiaoyuan Ding
+davidvrba - David Vrba
+deepyaman - Deepyaman Datta
+denglingang - Lingang Deng
+dengziming - dengziming
```
Comment (cloud-fan) [inline dev/create-release/known_translations:320]: @dengziming

File: dev/create-release/known_translations [hunk 17]
```diff
@@ -261,3 +261,151 @@ yanlin-Lynn - Yanlin Wang
...
+cchung100m - Neo Chien
+cclauss - Christian Clauss
+chakravarthiT - Chakravarthi
+chandulal - Chandu Kavar
+chitralverma - Chitral Verma
+cjn082030 - Jenny
+cloud-fan - Wenchen Fan
+codeborui - codeborui
+colinmjj - Colin Ma
+cxzl25 - cxzl25
```
Comment (cloud-fan) [inline dev/create-release/known_translations:313]: @cxzl25 can you let us know your full name?

File: dev/create-release/known_translations [hunk 18]
```diff
@@ -261,3 +261,151 @@ yanlin-Lynn - Yanlin Wang
...
+bettermouse - Chen Hao
+bscan - Brian Scannell
+cchung100m - Neo Chien
+cclauss - Christian Clauss
+chakravarthiT - Chakravarthi
+chandulal - Chandu Kavar
+chitralverma - Chitral Verma
+cjn082030 - Jenny
+cloud-fan - Wenchen Fan
+codeborui - codeborui
```
Comment (cloud-fan) [inline dev/create-release/known_translations:311]: @codeborui can you let us know your full name?

File: dev/create-release/known_translations [hunk 19]
```diff
@@ -261,3 +261,151 @@ yanlin-Lynn - Yanlin Wang
...
+bartosz25 - Bartosz Konieczny
+beliefer - Jiaan Geng
+bettermouse - Chen Hao
+bscan - Brian Scannell
+cchung100m - Neo Chien
+cclauss - Christian Clauss
+chakravarthiT - Chakravarthi
+chandulal - Chandu Kavar
+chitralverma - Chitral Verma
+cjn082030 - Jenny
```
Comment (cloud-fan) [inline dev/create-release/known_translations:309]: @cjn082030 can you let us know your full name?

File: dev/create-release/known_translations (hunk 11 above)
Comment (roland-ondeviceresearch) [reply dev/create-release/known_translations:374]: Roland Pogonyi

Comment (roland1982) [reply dev/create-release/known_translations:374]: Roland Pogonyi

File: dev/create-release/known_translations (hunk 17 above)
Comment (cxzl25) [reply dev/create-release/known_translations:313]: Shaoyun Chen

**Comment by gatorsmile** [comment]
After the release note is published, we still can update the contributor names. Please provide your full name. Thanks!

File: dev/create-release/known_translations (hunk 14 above)
Comment (eatoncys) [reply dev/create-release/known_translations:325]: Yanshan Chen

File: dev/create-release/known_translations (hunk 4 above)
Comment (wangjiaochun) [reply dev/create-release/known_translations:401]: Jiaochun Wang @cloud-fan

File: dev/create-release/known_translations (hunk 5 above)
Comment (wackxu) [reply dev/create-release/known_translations:400]: Shiwei Xu

**Comment by cjn082030** [comment]
@cloud-fan &nbsp; Juanni Chen





------------------ 原始邮件 ------------------
发件人:&nbsp;"Wenchen Fan"<notifications@github.com&gt;;
发送时间:&nbsp;2020年6月19日(星期五) 凌晨0:20
收件人:&nbsp;"apache/spark"<spark@noreply.github.com&gt;;
抄送:&nbsp;"daily"<1913870153@qq.com&gt;;"Mention"<mention@noreply.github.com&gt;;
主题:&nbsp;Re: [apache/spark] [MINOR] update dev/create-release/known_translations (#28861)





 
@cloud-fan commented on this pull request.
 
 
In dev/create-release/known_translations:
 &gt; +adrian555 - Weiqiang Zhuang +ajithme - Ajith S +amanomer - Aman Omer +ancasarb - Anca Sarb +avkgh - Aleksandr Kashkirov +ayudovin - Artsiom Yudovin +bartosz25 - Bartosz Konieczny +beliefer - Jiaan Geng +bettermouse - Chen Hao +bscan - Brian Scannell +cchung100m - Neo Chien +cclauss - Christian Clauss +chakravarthiT - Chakravarthi +chandulal - Chandu Kavar +chitralverma - Chitral Verma +cjn082030 - Jenny  
@cjn082030 can you let us know your full name?

File: dev/create-release/known_translations (hunk 19 above)
Comment (cjn082030) [reply dev/create-release/known_translations:309]: Juanni Chen

File: dev/create-release/known_translations (hunk 9 above)
Comment (teeyog) [reply dev/create-release/known_translations:393]: Yong Tian

File: dev/create-release/known_translations (hunk 10 above)
Comment (shahidki31) [reply dev/create-release/known_translations:382]: Shahid K I

File: dev/create-release/known_translations (hunk 6 above)
Comment (ulysses-you) [reply dev/create-release/known_translations:398]: @cloud-fan  Xiduo You

File: dev/create-release/known_translations (hunk 16 above)
Comment (dengziming) [reply dev/create-release/known_translations:320]: Ziming Deng

**Pull request closed without merging**